flair/datasets/sequence_labeling.py

```diff
@@ -665,6 +667,54 @@ def __init__(
             **corpusargs,
         )
 
+
+
+class WEBPAGES_NER(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "ner",
+            in_memory: bool = True,
+            **corpusargs,
+    ):
+        """
+               Initialize the Webpages corpus. This is only possible if you've manually downloaded it to your machine.
+               Obtain the corpus from https://cogcomp.seas.upenn.edu/Data/NERWebpagesColumns.tgz, merges them into one file and transform the format to an IOB format and put the webpage_ner IOB
+               files in a folder called 'webpage-ner'. Then set the base_path parameter in the constructor to the path to the
+               parent directory where the webpages folder resides.
+               :param base_path: Path to the webpages corpus (i.e. 'webpage-ner' folder) on your machine
+               :param tag_to_bioes: NER by default, need not be changed, but you could also select 'pos' or 'np' to predict
+               POS tags or chunks respectively
+               :param in_memory: If True, keeps dataset in memory giving speedups in training.
+               :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+               """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
```

Comment:
Can you add a routine here that downloads and unpacks the tgz file? The resulting files should then be concatenated into a single file for reading.